query-node/schemas/workingGroups.graphql

```diff
@@ -34,6 +34,9 @@ type Worker @entity {
   "The group that the worker belongs to"
   group: WorkingGroup!
 
+  "The id the group that the worker belongs to"
+  groupId: ID!
```

Comment:
This PR creates `groupId` attribute, but never explicitely uses it. It reIies on internal mechanics of Hydra that sets up this field automatically, I'm not mistaken. We could add comment about that.

```suggestion
  # This exposes internal Hydra value related to `group`. It will be internally set and updated by Hydra.
  # See https://github.com/Joystream/joystream/pull/3043 for more info.
  "The id the group that the worker belongs to"
  groupId: ID!
```